.github/workflows/R-CMD-check.yaml

```diff
@@ -102,3 +106,36 @@ jobs:
           name: ${{ runner.os }}-r${{ matrix.config.r }}-rust${{ matrix.config.rust-version }}-results
           path: check
 
+
+      - name: Test package generation
+        env:
+          _R_CHECK_CRAN_INCOMING_REMOTE_: false
+        run: |
+          remotes::install_local()
+          temp_dir <- tempdir()
+          pkg_dir <- file.path(temp_dir, "testpkg")
+          dir.create(pkg_dir)
+          setwd(pkg_dir)
+          devtools::create(".")
+          usethis::use_mit_license()
+          usethis::use_testthat()
+          brio::write_lines(
+              c(
+                  "test_that(\"`hello_world()` works\", {",
+                  "  expect_equal(hello_world(), \"Hello world!\")",
+                  "})"
+              ),
+              file.path("tests", "testthat", "test-hello.R")
+          )
+          rextendr::use_extendr()
+          devtools::document()
+          devtools::install()
```

Comment:
I think you can use `devtools::install(quick = TRUE)` here.

Reply:
I fixed this.